active_directory_ldap/komand_active_directory_ldap/util/utils.py

```diff
@@ -44,7 +60,40 @@ def dn_escape_and_split(dn):
         return dn_list
 
     @staticmethod
-    def find_parentheses_pairs(query_string):
+    def find_search_base(dn_list: list) -> str:
+        """
+        This method will find a search base from a dn_list
+        :param dn_list:
+        :return: Will return a properly formatted search base
+        """
+        dc_list = [s for s in dn_list if 'DC' in s]
+        search_base = ','.join(dc_list)
+        return search_base
+
+    def format_dn(self, dn: str) -> tuple:
```

Comment:
```suggestion
    def format_dn(self, dn: str) -> tuple:
```
Can you type the tuple return here? Example, if it returns a string tuple it would be `(str, str)` or for a string/int tuple, `(str, int)`